test_scripts/API/DeleteCommand/ATF_DeleteCommand_UNSUP_RES_to_VR_DelCmd_WARNINGS_to_UI_DelCmd.lua

```diff
@@ -0,0 +1,167 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
```

Comment:
@istoimenova , script was updated in commit  d2baa99d69379db10009b64a0949b062a9e6933f